### DISCUSSION TITLE
421

### DISCUSSION THREAD

**Discussion by giscus** [Shane-Park/markdownBlog#166] [Comments]
# 421

Intro 자바에서도 QClass 생성은 처음 하면 꽤나 당황스러운 과정인데, 코틀린은 또 과정이 달랐기에 결과물을 기록으로 남기고자 합니다. 2022년 9월 19일 기준의 최신 라이브러리들에서 잘 작동하고 있는 방법 입니다. 설정 build.gradle.kts plugins { ... kotlin("kapt") version "1.7.10" } dependencies { ... implementation("com.querydsl:querydsl-jpa:5.0.0") kapt("com.querydsl:querydsl-apt:5.0.0:jpa") } 소스코드는 딱 위의 변경 사항만 있었습니다. 의존성 변경 후에 Shift + Command + I 키를 입력 해서 Load Gradle Changes를 해 주..

https://shanepark.tistory.com/421

**Comment by ghdcksgml1** [comment]
저는 이방법으로는 안돼서 이렇게 해결했습니다.

//querydsl 추가
implementation("com.querydsl:querydsl-jpa:5.0.0:jakarta")
implementation("com.querydsl:querydsl-core:5.0.0")
kapt("com.querydsl:querydsl-apt:5.0.0:jakarta")
kapt("com.querydsl:querydsl-kotlin-codegen:5.0.0")
